python/sdss_access/path/path.py

```diff
@@ -1125,13 +1125,66 @@ def healpixgrp(self, filetype, **kwargs):
         Returns
         -------
         healpixgrp : str
-            HEALPix group directory, HEALPix//100.
+            HEALPix group directory, HEALPix//1000.
 
         '''
 
         healpix = int(kwargs['healpix'])
-        subdir = "{:d}".format(healpix//100)
+        subdir = "{:d}".format(healpix//1000)
         return subdir
 
+    def apgprefix(self, filetype, **kwargs):
+        ''' Returns APOGEE prefix using telescope/instrument.
+
+        Parameters
+        ----------
+        filetype : str
+            File type parameter.
+        telescope : str
+            The APOGEE telescope (apo25m, lco25m, apo1m).
+        instrument : str
+            The APOGEE instrument (apogee-n, apogee-s).
+
+        Returns
+        -------
+        prefix : str
+            The APOGEE prefix (ap/as).
+
+        '''
+
+        telescope = kwargs.get('telescope')
```

Comment:
To ensure that telescope is set to None if it is not present in the kwargs, try `telescope = kwargs.get('telescope', None)`.  Otherwise this will throw an uncaught error.